internal/driver/webhtml.go

```diff
@@ -690,6 +707,11 @@ function viewer(baseUrl, nodes) {
     toptable.addEventListener("touchstart", handleTopClick)
   }
 
+  for (const id of ["topbtn", "graphbtn", "peek", "list", "disasm",
```

Comment:
at this point it seems that these arrays (also e.g. line 690) would benefit from being extracted to a constant as they may be tedious to update consistently.
There is a nicer way to make this loop as
`ids.forEach(makeLinkDynamic)`.

There is a potential JS trap here if written as `["focus", "ignore", "hide", "show"].forEach(makeLinkDynamic)` it may fail if the code before is an expression, since it will parse it as indexing of the result value instead of an array literal.
In that case for the semicolon-free JS a semicolon would be needed after all.

Reply:
Done.